### PR TITLE
[ARGV] Allow specifying an option multiple times

### PR DESCRIPTION
**Pull request by segiddins**
\c @kylef @alloy

### CODE REVIEW

File: spec/argv_spec.rb [hunk 1]
```diff
@@ -53,6 +53,13 @@ module CLAide
         @argv.remainder.should == %w(--flag ARG1 ARG2 --no-other-flag)
       end
 
+      it 'allows mutliple values for the same option' do
+        @argv = ARGV.new %w(--ignore=foo --ignore=bar)
+        @argv.option('ignore').should == 'bar'
+        @argv.option('ignore').should == 'foo'
+        @argv.option('ignore').should.be.nil
+      end
```
Comment (alloy) [inline spec/argv_spec.rb]: How is this gonna be used? Would it not be more useful if there were a variant that would return an array of all values for 1 option key?

Comment (segiddins) [inline spec/argv_spec.rb]: I'm switching Xcodeproj to CLAide, and I needed a way to allow ignoring multiple things, and I wanted to do it in the most minimally-invasive way possible. I could add a new help to return all options for a given value?

Comment (alloy) [inline spec/argv_spec.rb]: Yeah I think adding one would be nice :+1:

**Comment by segiddins** [comment]
@alloy helper added

**Comment by alloy** [comment]
Purrfect, you da man :+1:

**Comment by segiddins** [comment]
:shipit:

**Pull request merged**